packages/template-retail-react-app/app/components/product-view/index.jsx

```diff
@@ -395,6 +395,16 @@ const ProductView = forwardRef(
             }
         }, [variant?.productId, quantity])
 
+        useEffect(() => {
+            if (
+                isProductPartOfBundle &&
+                product &&
+                (!product.variationAttributes || product.variationAttributes.length === 0)
+            ) {
```

Comment:
Minor: if we're checking only for standard product type here, we can check for `product.type.item` instead.

Reply:
Hi @sf-emmyzhang, if you haven't already do you mind creating a small PR to address this comment from @vmarta. Thank you!